index.js

```diff
@@ -37,7 +60,7 @@ const displayBooks = () => {
     removeBtn.classList.add(`remove-${book.id}`);
     removeBtn.textContent = 'Remove';
     removeBtn.addEventListener('click', () => {
-      removeBook(book.id);
+      methods.removeBook(book.id);
```

Comment:
- [ ] The other functionality that does not have anything to do with the functionality of the books, but with the DOM manipulation can go inside a `Display` class, this way the code is will be more readable 😄